lib/smart_proxy_reports/friendly_message.rb

```diff
@@ -12,7 +12,12 @@ def generate_message
     return @result_tree["msg"] if @result["failed"]
 
     case @task_tree["action"]
+    when "ansible.builtin.assemble", "assemble" then msg = assemble_message
+    when "ansible.builtin.dnf", "dnf" then msg = dnf_message
+    when "ansible.builtin.git", "git" then msg = git_message
     when "ansible.builtin.package", "package" then msg = package_message
+    when "ansible.builtin.systemd", "systemd" then msg = systemd_message
+    when "ansible.builtin.tempfile", "tempfile" then msg = tempfile_message
```

Comment:
Could we also add here a regular `file` (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#ansible-collections-ansible-builtin-file-module) as well? I guess it's being pretty used.

Reply:
Yeah please add that thanks.